build_library/build_image_util.sh

```diff
@@ -154,6 +154,15 @@ write_packages() {
     fi
 }
 
+extract_docs() {
+    local root_fs_dir="$1"
+
+    info "Extracting docs"
+    tar --create --bzip --file="${BUILD_DIR}/docs.tar" \
```

Comment:
Doesn't this need to be `tar -caf "${BUILD_DIR}/doc.tar.bz2" -C "${root_fs_dir}/usr/share/coreos" doc`? The auto-compress flag isn't implied by default.